firewall_with_target_tags/main.tf

```diff
@@ -1,6 +1,5 @@
 # [START vpc_firewall_create]
 resource "google_compute_firewall" "rules" {
-  project     = "my-project-name"
```

Comment:
Should we add:

```
data "google_project" "project" {
}
```

Reply:
Similar to above, we can continue the conversation there.